frontend/src/libs/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import Select, { type StylesConfig } from 'react-select';
+
+import { useCallback, useMemo, useState } from '~/libs/hooks/hooks.js';
+
+type Option = { label: string; value: string };
```

Comment:
Let's move it to `~/libs/types` and rename to SelectOption. We'll need it in case we want to declare options in page domain.